firmware/layout2.c

```diff
@@ -419,6 +420,24 @@ void layoutSignIdentity(const IdentityType *identity, const char *challenge)
 		NULL);
 }
 
+void layoutSignEcdsa(const SignEcdsa *signEcdsa)
```

Comment:
This displays the SHA256 of the data being signed in order for the user to verify they're signing what they expect.